tools/gluten-te/centos/defaults.conf

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -ex
+
+## For basic scripts
+
+# Non-interactive during docker run
+DEFAULT_NON_INTERACTIVE=OFF
+
+# Do not remove stopped docker container
+DEFAULT_PRESERVE_CONTAINER=OFF
+
+# The codes will be used in build
+DEFAULT_GLUTEN_REPO=https://github.com/oap-project/gluten.git
+DEFAULT_GLUTEN_BRANCH=main
+
+# Create debug build
+DEFAULT_DEBUG_BUILD=OFF
+
+# Backend type (velox)
+DEFAULT_BUILD_BACKEND_TYPE=velox
+
+# HTTP proxy
+DEFAULT_HTTP_PROXY_HOST=
+DEFAULT_HTTP_PROXY_PORT=
+
+# If on, use maven mirror settings for PRC's network environment
+DEFAULT_USE_ALI_MAVEN_MIRROR=OFF
+
+# Base operator system image used in build scripts.
+DEFAULT_OS_IMAGE=centos:8
+
+# Set timezone name
+DEFAULT_TIMEZONE=Asia/Shanghai
+
+# Build will result in this image
+DEFAULT_DOCKER_TARGET_IMAGE_BUILDENV=gluten-te/centos/gluten-buildenv
+
+# Build will result in this image
+DEFAULT_DOCKER_TARGET_IMAGE_BUILD=gluten-te/centos/gluten-build
+
+# Docker cache image used to speed-up builds
+DEFAULT_DOCKER_CACHE_IMAGE=
+
+## For tpc.sh
+
+# Java options
+DEFAULT_EXTRA_JAVA_OPTIONS="-Xmx2G"
+
+# Run GDB.
+DEFAULT_RUN_GDB=OFF
+
+# Run GDB server.
+DEFAULT_RUN_GDB_SERVER=OFF
+
+# GDB server bind port
+DEFAULT_GDB_SERVER_PORT=2345
+
+# Run JVM jdwp server.
+DEFAULT_RUN_JDWP_SERVER=OFF
+
+# JVM jdwp bind port
+DEFAULT_JDWP_SERVER_PORT=5005
+
+# Docker options
+DEFAULT_EXTRA_DOCKER_OPTIONS="--network bridge"
+
+# Build will result in this image
+DEFAULT_DOCKER_TARGET_IMAGE_TPC=gluten-te/centos/gluten-tpc
+DEFAULT_DOCKER_TARGET_IMAGE_TPC_GDB=gluten-te/centos/gluten-tpc-gdb
+DEFAULT_DOCKER_TARGET_IMAGE_TPC_GDB_SERVER=gluten-te/centos/gluten-tpc-gdb-server
```

Comment:
Can we remove the unused options? Only some of them are useful for cbash.sh

Reply:
Yes, I will remove it.